src/components/newPayment/PaymentDetailsForm.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  DIcon,
+  DButton,
+  DInputCheck,
+  DInputSelect,
+  DInput,
+  useDToast,
+  useDPortalContext,
+} from '@dynamic-framework/ui-react';
+import { useFormik } from 'formik';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import * as Yup from 'yup';
+
+import { Bill } from '../../services/interface';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { getPayDates, getSelectedCompany, getSelectedService } from '../../store/selectors';
```

Comment:
Añade la regla de eslint que está en el base-template. Para que te tire los imports 
```suggestion
import { 
  getPayDates, 
  getSelectedCompany, 
  getSelectedService
} from '../../store/selectors';
```